Duplicati/UnitTest/ProblematicPathTests.cs

```diff
@@ -96,7 +88,7 @@ public void DirectoriesWithWildcards()
                     {
                         string fileName = SystemIO.IO_OS.PathGetFileName(expectedFilePath);
                         string restoredFilePath = SystemIO.IO_OS.PathCombine(this.RESTOREFOLDER, directoryName, fileName);
-                        Assert.IsTrue(TestUtils.CompareFiles(expectedFilePath, restoredFilePath, expectedFilePath, false));
+                        TestUtils.AssertFilesAreEqual(expectedFilePath, restoredFilePath, false, expectedFilePath);
                     }
```

Comment:
Is it possible to replace this `foreach` loop with a call to `AssertDirectoryTreesAreEquivalent`?